hooks/environment

```diff
@@ -75,24 +73,32 @@ plugin_read_list_into_result() {
     [[ ${#result[@]} -gt 0 ]] || return 1
 }
 
-envconsul_env() {
-    # This populates a `result` array for later use
-    plugin_read_list_into_result BUILDKITE_PLUGIN_VAULT_ENV_SECRETS
+secrets=()
+if plugin_read_list_into_result BUILDKITE_PLUGIN_VAULT_ENV_SECRETS; then
+    secrets=("${result[@]}")
+else
+    raise_error "At least one secret must be specified!"
+fi
 
-    secrets=()
-    for secret in "${result[@]}"; do
+########################################################################
+
+readonly container_name="vault-env-plugin-${BUILDKITE_JOB_ID}"
+
+envconsul_env() {
+    args=()
+    for secret in "${secrets[@]}"; do
         # secret_prefix is guaranteed to end with a / if it is non-empty
-        secrets+=("-secret=${secret_prefix}${secret}")
+        args+=("-secret=${secret_prefix}${secret}")
```

Comment:
nit: since this does contain secrets, I would prefer that the variable name made that clear. Maybe we prefix this with secrets ala secret_args?

Reply:
I'll update the name, but these are still just the _names_ of the secrets to pull from Vault; they aren't secrets themselves.